### ISSUE TITLE
Referrer policy and Ping-From/Ping-To headers

### ISSUE THREAD

**Issue by annevk** [w3c/webappsec#469]
HTML defines these two headers and says at least `Ping-From` is subject to the same rules as `Referer` in a somewhat roundabout way. Presumably at some point we want to define this accurately, but that would mean that the Referrer standard needs to provide some hooks.

**Comment by mikewest** [comment]
Makes sense that we'd tie these together: throwing this at @jeisinger.

**Comment by jeisinger** [comment]
do you mean that the Ping-From should respect the referrer policy or that the ping spec should state which referrer policy should be applied (probably default)?

**Comment by annevk** [comment]
I think it should respect the policy, otherwise `ping` would leak data `href` wouldn't. It seems easiest if the `ping` specification could invoke some hook in the Referrer Policy to determine what the header value should be, but I'm open to alternative setups.

**Comment by jeisinger** [comment]
yeah, unless the policy is always and previously, ping-from would have been empty. But that sounds unlikely.

**Comment by annevk** [comment]
(The current specification for `ping` already limits the value in various ways based what happens to referrer. That's why this came up. E.g., not sending `Ping-From` for downgrades.)

**Comment by jeisinger** [comment]
@estark37
